fastlane_core/lib/fastlane_core/cert_checker.rb

```diff
@@ -81,16 +81,21 @@ def self.list_available_developer_id_installer(in_keychain: nil)
       `#{commands.join(' ')}`
     end
 
-    def self.wwdr_certificate_installed?
+    def self.wwdr_certificates_installed?
       certificate_name = "Apple Worldwide Developer Relations Certification Authority"
       keychain = wwdr_keychain
-      response = Helper.backticks("security find-certificate -c '#{certificate_name}' #{keychain.shellescape}", print: FastlaneCore::Globals.verbose?)
-      return response.include?("attributes:")
+      response = Helper.backticks("security find-certificate -a -c '#{certificate_name}' #{keychain.shellescape}", print: FastlaneCore::Globals.verbose?)
+      certs = response.split("keychain: \"#{keychain}\"").drop(1)
+      certs.count == 2
     end
 
-    def self.install_wwdr_certificate
-      url = 'https://developer.apple.com/certificationauthority/AppleWWDRCA.cer'
-      file = Tempfile.new('AppleWWDRCA')
+    def self.install_wwdr_certificates
+      install_wwdr_certificate('https://developer.apple.com/certificationauthority/AppleWWDRCA.cer')
```

Comment:
Do we actually need this one as well? Why?

Reply:
It doesn't expire until 3 more years, and there are many, many certificates issued by it.

Reply:
Oh I thought it was expiring by Feb 2021 for some reason 🤔 

All good 👍 thanks for explaining